repository/src/main/java/org/alfresco/repo/content/ContentServiceImpl.java

```diff
@@ -612,7 +605,7 @@ public boolean isContentDirectUrlEnabled(NodeRef nodeRef)
                 throw new IllegalArgumentException("The supplied nodeRef " + nodeRef + " has no content.");
             }
 
-            contentDirectUrlEnabled = (store.isContentDirectUrlEnabled(nodeRef));
+            contentDirectUrlEnabled = (store.isContentDirectUrlEnabled(getContentUrl(nodeRef)));
```

Comment:
Minor: Generally it is better to not nest method calls to simplify debug. However leave it this time.

Reply:
Fair point. Thanks